### ISSUE TITLE
Allow mirroring on different groups

### ISSUE THREAD

**Issue by Soullivaneuh** [samrocketman/gitlab-mirrors#73]
Hi and thanks for this project.

If I well understand the documentation, we have to create a "Mirrors" group and set up project to mirror that will be auto added on this group.

Could be very useful to have the possibility to set group when adding a mirror.

For example:

```bash
./add_mirror.sh --git --project-group samrocketman --project-name gitlab-mirrors --mirror https://github.com/samrocketman/gitlab-mirrors.git
```

Thanks.

**Comment by samrocketman** [comment]
Yeah, currently the work around is to have multiple copies of gitlab-mirrors.  I'd say it's an easy enough feature that I can add that :)

**Comment by Soullivaneuh** [comment]
Glad to hear it, can't wait to see this feature coming to use your tool! :+1: 

I'm sorry I can't work on a PR, I'm not very familiar with Python... ;)

**Comment by samrocketman** [comment]
It's not python, it's all bash.  The only python bits are to interact with the GitLab API.

**Comment by Soullivaneuh** [comment]
Hmm had you plan to make this feature soon ? If not, I will take a look for a PR...

**Comment by samrocketman** [comment]
I work on the project whenever in my spare time.

**Comment by samrocketman** [comment]
That is to say I don't want to commit to a timeline.  But I will definitely get around to create it even if you don't submit a PR.  Feel free to submit a PR or you can wait until I get around to it.

**Comment by samrocketman** [comment]
Possibly related #102

**Comment by Facinorous-420** [comment]
I created a repo that makes it easy to manage multiple groups. Note, my repo does clone this one for the main files **but** edits the python file to add automation to group creation. You can check out the repo here -> https://github.com/Facinorous-420/gitmirrors-bs

I'll try to keep it updated and working as this one gets updated.
